src/MakingSense.AspNetCore.Documentation/BaseDocumentHandler.cs

```diff
@@ -20,10 +20,25 @@ public abstract class BaseDocumentHandler : IDocumentHandler
 
 		public int StatusCode => StatusCodes.Status200OK;
 
-		public BaseDocumentHandler(IFileProvider fileProvider, string subpath)
+		public BaseDocumentHandler(IFileProvider fileProvider, string subpath, List<string> languages)
```

Comment:
Why is it necessary a list of languages? is not enough with one of them?

Also, I thought about this resolution:

```
/es/aaaa => aaaa.es (and if not exist) => aaaa
/en/aaaa => aaaa.en (and if not exist) => aaaa
/aaaa => aaaa.en (and if not exist) => aaaa
```

But it is working in a different way, right?

Reply:
Yes, I was thinking on try the input language first, then the default one, and finally the default file.

In case `en` is defined as default:

`/es/aaaa => aaaa.es (and if not exist) => aaaa.en (and if not exist) => aaaa`
`/en/aaaa => aaaa.en (and if not exist) => aaaa`
`/aaaa => aaaa.en (and if not exist) => aaaa`

If this is not necessary, we could avoid using the language list.

Reply:
I think that we could use default language only in URL resolution. For example, if the default language is `en` the URLs `/en/aaaa` and `/aaaa` will have similar results.

Then, in the resolution, we could be agnostic about default language, we ask for the _.en_ file, and if it does not exist, we could use the file without language extension.

I think that it is simpler and cover all scenarios, let me know if I am missing something.